server/tserver/src/main/java/org/apache/accumulo/tserver/tablet/ScanDataSource.java

```diff
@@ -192,8 +192,8 @@ public boolean isCurrent() {
     MultiIterator multiIter = new MultiIterator(iters, tablet.getExtent());
 
     TabletIteratorEnvironment iterEnv = new TabletIteratorEnvironment(
-        tablet.getTabletServer().getContext(), IteratorScope.scan, tablet.getTableConfiguration(),
-        fileManager, files, options.getAuthorizations(), samplerConfig);
+        tablet.getTabletServer().getContext(), IteratorScope.scan, fileManager, files,
```

Comment:
This used to pass the table config, now it seems the use has no way to get that.

Reply:
Yeah I will revert this.  I wasn't sure which impl was used on the server side when I was removing code.